### DISCUSSION TITLE
ValueError: GPU is not accessible. Was the library installed correctly?

### DISCUSSION THREAD

**Discussion by ken-delphai** [explosion/spaCy#10840] [Help: Installation]
```
  File "/opt/homebrew/Caskroom/miniforge/base/envs/ner/lib/python3.10/site-packages/thinc/util.py", line 212, in require_gpu
    raise ValueError("GPU is not accessible. Was the library installed correctly?")
```

Unable to train the Chinese NER model using GPU on M1 MacBook Pro 14. Did I install spaCy incorrectly? I followed the instruction and installed spacy[apple]

**Comment by polm** [comment]
Sorry, the GPU on M1 Macs is not supported right now, see [here](https://github.com/explosion/spaCy/issues/10783#issuecomment-1132523032)
